drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorViewImpl.java

```diff
@@ -124,6 +129,10 @@ public void execute() {
                            );
     }
 
+    private String getClientId() {
+        return ( (ClientMessageBusImpl) clientMessageBus ).getClientId();
```

Comment:
cast to busimpl should not be needed. getClientId is on the ClientMessageBus interface, no?